test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -7794,10 +7793,8 @@ def write_resourcedef(self, resources=None, filename=None, restart=True):
                                 'resourcedef')
         else:
             dest = filename
-        self.du.run_copy(self.hostname, fn, dest, mode=0644, sudo=True)
-        if filename is None:
-            self.du.chown(self.hostname, path=dest, uid=0, gid=0,
-                          sudo=True)
+        self.du.run_copy(self.hostname, fn, dest, sudo=True,
+                         preserve_permission=False)
```

Comment:
can we make 'preserve_permission' default to False? Because that will be mostly used.